definitions/npm/@testing-library/react_v10.x.x/flow_v0.104.x-/react_v10.x.x.js

```diff
@@ -254,18 +272,8 @@ declare module '@testing-library/react' {
     |}
   ): Promise<T>;
 
-  declare export function waitForElementToBeRemoved(
-    callback?: HTMLElement,
-    options?: {|
-      container?: HTMLElement,
-      timeout?: number,
-      interval?: number,
-      mutationObserverOptions?: MutationObserverInit,
-    |}
-  ): Promise<HTMLElement>;
-
   declare export function waitForElementToBeRemoved<T>(
-    callback?: () => T,
+    callback: (() => T) | T,
```

Comment:
Humm okay, so if you pass an HTML Element as `callback` it's properly inferred? 
https://testing-library.com/docs/dom-testing-library/api-async#waitforelementtoberemoved